features/references/manage_working_patterns.feature

```diff
@@ -0,0 +1,57 @@
+Feature: Identify Judicial Office Holder Working Patterns and allocate to hearing that they are working and available
+
+  As a Solution
+  I want to be able to identify a judicial office officer's daily and weekly working pattern.
+  So that the hours worked each day and the days of the week worked can be applied to the scheduling and listing of hearings.
+
+  Background:
+    Given Multi-Day Hearing is created
```

Comment:
You would also need a single full day and single partial day hearing

Reply:
Updated the background to have a single and partial day hearing